org/cloudfoundry.yml

```diff
@@ -3405,7 +3418,6 @@ orgs:
           bosh-bbl-ci-envs: admin
           bosh-cli: admin
           bosh-compiled-releases-index: admin
-          bosh-cpi-certification: admin
```

Comment:
To be propagated: https://github.com/cloudfoundry/community/commit/2f51e7779d0d48135b406c1051094a13954cc99d